src/boot/boot.c

```diff
@@ -241,7 +241,7 @@ static char16_t* update_timeout_efivar(Config *config, bool inc) {
         case TIMEOUT_MENU_FORCE:
                 return xstrdup16(u"Timeout disabled, menu will always be shown.");
         case TIMEOUT_MENU_HIDDEN:
-                return xstrdup16(u"Menu hidden. Hold down key at bootup to show menu.");
+                return xstrdup16(u"Menu hidden. Hold down space at bootup to show menu.");
```

Comment:
Hmm, but sd-boot maybe used in tablet or smartphone like devices, which may not have space, but may have e.g. volume-key or so.

Reply:
I mean, I agree with that space key is suitable for most cases, but saying 'space' in the message may confuse some people.

Reply:
I'll drop this commit then.